src/fastga_he/models/performances/mission_vector/mission/equilibrium_alpha.py

```diff
@@ -70,10 +79,10 @@ def setup(self):
             of="alpha",
             wrt=[
                 "data:geometry:wing:area",
-                "data:aerodynamics:wing:cruise:CL0_clean",
-                "data:aerodynamics:wing:cruise:CL_alpha",
-                "data:aerodynamics:horizontal_tail:cruise:CL0",
-                "data:aerodynamics:horizontal_tail:cruise:CL_alpha",
+                "data:aerodynamics:wing:" + ls_tag + ":CL0_clean",
```

Comment:
For components we rework or add, the declaration of partials should always be in a separate `setup_partials `function. See this example: https://openmdao.org/newdocs/versions/latest/features/core_features/working_with_derivatives/specifying_partials.html